src/viam/ur/module/ur_arm.cpp

```diff
@@ -404,6 +403,9 @@ class URArm::state_ {
         using state_event_handler_base_<state_independent_>::handle_event;
 
         reason reason_;
+
+        // track whether we have attempted to reconnect to local mode to avoid being spammy about it.
+        bool logged_local_reconnect_attempt{false};
```

Comment:
added this so we were less spammy during local mode. 

disconnected mode is also pretty spammy, so it might be worth doing something similar there? the factory will turn off the arm at EOD